integration-test-groups/foundation/language/pom.xml

```diff
@@ -43,6 +47,37 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy</artifactId>
         </dependency>
+        
+        <!-- Non-core languages -->
+        <dependency>
```

Comment:
We have faced situations where separating tests in their own project helped, as sometime an extension is hiding a bug from another one.

Here, it might be ok to bundle everything as the main purpose is not really to test each language, but more to test that each language can be loaded in Camel Quarkus.